src/integration-tests/src/showcase.rs

```diff
@@ -0,0 +1,168 @@
+        .with_request(request.clone())
+        .send()
+        .await?;
+    tracing::info!("response: {response:?}");
+    assert_eq!(response.binding_uri, "/v1beta1/repeat:body");
+    assert_eq!(response.request, Some(request));
+
+    let request = RepeatRequest::new()
+        .set_f_int32(1)
+        .set_f_int64(2)
+        .set_f_double(3)
+        .set_p_int32(4)
+        .set_p_int64(5)
+        .set_p_double(6.5)
+        .set_info(
+            ComplianceData::new()
+                .set_f_string("the quick brown fox jumps over the lazy dog")
+                .set_f_int32(1)
+                .set_f_sint32(2)
+                .set_f_sfixed32(3)
+                .set_f_uint32(4_u32)
+                .set_f_fixed32(5_u32)
+                .set_f_int64(6)
+                .set_f_sint64(7)
+                .set_f_sfixed64(8)
+                .set_f_sfixed64(8)
+                .set_f_uint64(9_u64)
+                .set_f_fixed64(10_u64)
+                .set_f_double(11.25_f64)
+                .set_f_float(12.5_f32)
+                .set_f_bool(true)
+                .set_f_bytes(bytes::Bytes::from_static(
+                    b"How vexingly quick daft zebras jump!",
+                ))
+                .set_f_kingdom(LifeKingdom::Fungi)
+                .set_f_child(
+                    ComplianceDataChild::new()
+                        .set_f_continent(Continent::Africa)
+                        .set_p_continent(Continent::Australia),
+                )
+                .set_p_string(
+                    "Answer to the Ultimate Question of Life, the Universe, and Everything"
+                        .to_string(),
+                )
+                .set_p_int32(42)
+                .set_p_double(42.42)
+                .set_p_bool(false)
+                .set_p_kingdom(LifeKingdom::Eubacteria),
+        );
+    let response = client
+        .repeat_data_body()
+        .with_request(request.clone())
+        .send()
+        .await?;
+    tracing::info!("response: {response:?}");
+    assert_eq!(response.binding_uri, "/v1beta1/repeat:body");
+    assert_eq!(response.request, Some(request));
+
+    Ok(())
+}
+
+async fn install() -> Result<()> {
+    let install = Command::new("go")
+        .args(["install", SHOWCASE_NAME])
+        .stdin(Stdio::null())
+        .output()
+        .await
+        .map_err(Error::other)?;
+    if !install.status.success() {
+        return Err(Error::other(format!(
+            "error installing showcase: {install:?}"
+        )));
+    }
+    tracing::info!("installed showcase binary: {install:?}");
+    Ok(())
+}
+
+async fn wait_until_ready() -> Result<()> {
+    let client = showcase::client::Testing::builder()
+        .with_endpoint("http://localhost:7469")
+        .with_credentials(auth::credentials::testing::test_credentials())
+        .with_retry_policy(gax::retry_policy::NeverRetry)
+        .with_tracing()
+        .build()
+        .await?;
+
+    for _ in 0..10 {
```

Comment:
Q: why the manual loop vs. gax's retry loop? is it just easier to write?

Reply:
Nah, I was just troubleshooting things and left the explicit loop in. Fixed.